crates/iota-data-ingestion/tests/grpc_blob_ingestion.rs

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Mysten Labs, Inc.
+// Modifications Copyright (c) 2025 IOTA Stiftung
```

Comment:
Same goes to all the new files.
```suggestion
// Copyright (c) 2025 IOTA Stiftung
```